dist/js/script.js

```diff
@@ -3,3 +3,49 @@
  * Date: 12 - 06 -2021
  * Copyright (c) 2021 Calvince
  */
+
+(function () {
+  // Utility functions
+
+  // /**
+  //  * @param {string} idValue - an element ID
+  //  * @returns {Object} DOM object associated with id
+  //  * */
+  // function id(idName) {
+  //   return document.getElementById(idName);
+  // }
+
+  /**
+   * @param {string} className - a classs name of the html elements
+   * @returns {Object} Dom object(s) of the associated html elements
+   */
+  function classItem(className) {
+    return document.querySelector(className);
+  }
+
+  // Variables
+  let isOpen = false;
+  const menu = classItem('.mobile-menu-button');
+  const menuItem = classItem('.mobile-menu-button-item');
+  const nav = classItem('.desktop-menu');
```

Comment:
- [ ]  **Please make sure that the user click (or tap) any option of the mobile menu, the mobile menu disappear. Also, make sure that the links are working**

This is the default behavior in mobile menus. As soon as a link is clicked, the menu should close and the user should be taken on the respective link. Please implement this.